tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/auto_query_fixer/QueryTokenService.java

```diff
@@ -0,0 +1,85 @@
+package com.google.cloud.bigquery.utils.auto_query_fixer;
+
+import org.apache.calcite.sql.parser.SqlParser;
+import org.apache.calcite.sql.parser.babel.SqlBabelParserImpl;
+import org.apache.calcite.sql.parser.babel.Token;
+import org.apache.calcite.util.SourceStringReader;
+import org.apache.commons.lang3.tuple.Pair;
+
+import com.google.cloud.bigquery.utils.auto_query_fixer.entity.IToken;
+import com.google.cloud.bigquery.utils.auto_query_fixer.entity.TokenImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class QueryTokenService {
```

Comment:
Please add comment for the class and each method according to the style guide. It would be harder to understand without comments :).